src/Hazelcast.Net/Networking/NetworkingOptions.cs

```diff
@@ -47,6 +47,16 @@ private NetworkingOptions(NetworkingOptions other)
             ConnectionRetry = other.ConnectionRetry.Clone();
         }
 
+        /// <summary>
+        /// Gets the default Hazelcast server port.
+        /// </summary>
+        public int DefaultPort { get; } = 5701;
```

Comment:
These two are not options. They are just internal consts.

Reply:
Fixed and pushed.